opencolorio_config_aces/config/cg/generate/config.py

```diff
@@ -204,7 +205,7 @@ def clf_transform_to_description(
             DescriptionStyle.SHORT_UNION,
         ):
             if clf_transform.description is not None:
-                if inverse_direction:
+                if direction == "Forward":
```

Comment:
maybe `if direction.lower() == "forward":` so that it becomes case insensitive?

LGTM otherwise and thank you!

Reply:
Updated with the case insensitive check.
Let me know of anything else that needs to be addressed!

Reply:
We are good! Thanks a lot @ajymoonILM!